tuf/client/updater.py

```diff
@@ -1142,12 +1142,10 @@ def neither_403_nor_404(mirror_error):
           # Otherwise, reraise the error, because it is not a simple HTTP
           # error.
           if neither_403_nor_404(mirror_error):
-            logger.exception('Misc error for root version '+str(next_version))
+            logger.info('Misc error for root version '+str(next_version) + str(exception))
             raise
           else:
-            # Calling this function should give us a detailed stack trace
-            # including an HTTP error code, if any.
-            logger.exception('HTTP error for root version '+str(next_version))
+            logger.debug('HTTP error for root version '+str(next_version))
```

Comment:
Nit: missing space between those two `'HTTP error for root version '+str(next_version)`